app/pages/terms-of-use.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Head from 'next/head';
+
+import { useRouter } from 'next/router';
+
+import Header from 'layout/header';
+import Footer from 'layout/footer';
+import MetaTags from 'layout/meta-tags';
+
+import Contact from 'layout/statics/contact';
+import PrivacyPolicyTerms from 'layout/statics/privacy-policy';
+
+import MARXAN_SOCIAL_MEDIA_IMG from 'images/social-media/marxan-social-media.png';
+
+import { withUser } from 'hoc/auth';
+
+export const getServerSideProps = withUser();
+
+const TermsOfUse: React.FC = () => {
+  const { asPath } = useRouter();
+
+  return (
+    <>
+      <Head>
+        <title>Terms of Use</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <MetaTags
+        name="Marxan conservation Solutions"
+        title="Terms of use"
+        description=""
+        url={`${process.env.NEXT_PUBLIC_VERCEL_URL}${asPath}`}
+        image={MARXAN_SOCIAL_MEDIA_IMG}
```

Comment:
With the last PR I merged related to MetaIcons, don't you think we need to keep the same changes in those 2 new pages? Pull from develop and fix it please

Reply:
Sure, done!